src/invidious/views/components/description_chapters_widget.ecr

```diff
@@ -0,0 +1,34 @@
+<% if !chapters.empty? %>
+    <div class="description-chapters-section">
+        <hr class="description-content-separator"/>
+        <h4><%=HTML.escape(translate(locale, "video_chapters_label"))%></h4>
+
+        <% if video.automatically_generated_chapters? %> 
+          <h5><%=HTML.escape(translate(locale, "video_chapters_auto_generated_label"))%> </h5>
+        <% end %>
+
+        <div class="description-chapters-content-container">
+            <% chapters.each do | chapter | %>
+                <%- start_in_seconds = chapter.start_ms.total_seconds.to_i %>
+                <a href="/watch?v=<%-= video.id %>&t=<%=start_in_seconds %>">
```

Comment:
The attributes `data-onclick="jump_to_time" data-jump-time="<seconds>"` should go on this `<a>` element. They don't need to go anywhere else!

Reply:
Also, we can pass a floating point number, with full ms precision, as the [function that decodes the `t` parameter](https://github.com/iv-org/invidious/blob/b16f66ef0003843c4561f7bb3124339e6b446b2b/src/invidious/helpers/utils.cr#L73) supports milliseconds precision (And on the JS side, [`player.currentTime()` does too](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/currentTime)).